k8s/gke/k8s-installer/vars.yml

```diff
@@ -0,0 +1,10 @@
+---
+user: "{{ lookup('env','GKEUSER') }}"
+initial_node_count: 3
+machine_type: n1-standard-2
+disk_size_gb: 80
+image_type: Ubuntu
+zone: us-central1-a
+project: openebs-ci
+auth_kind: serviceaccount
+
```

Comment:
Would it be good to accept kubernetes version with defaults ? Unlike other platforms, GKE has its own versions created on top of kubernetes versions. Which may need us to explicitly specify supported versions OR construct the gke version based on user's general k8s version input

Reply:
@ksatchit , the gcp_container_cluster module currently does not have an attribute to accept the k8s version. It takes the default version that GKE provides when creating a cluster. Will add it when the module starts supporting the version attribute.